core/docker/Dockerfile

```diff
@@ -11,7 +11,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-FROM eclipse-temurin:17.0.4_8-jdk
+FROM eclipse-temurin:17-jdk
+
+RUN \
+    set -xeu && \
+    echo 'Acquire::Retries "3";' > /etc/apt/apt.conf.d/80-retries && \
+    echo 'Acquire::http::Timeout "15";' > /etc/apt/apt.conf.d/80-timeouts && \
+    apt-get update -q && \
+    apt-get install -y -q git gcc make && \
+    git clone https://github.com/airlift/jvmkill /tmp/jvmkill && \
```

Comment:
Should we pin this to some commit? I expect jvmkill to rarely change but still it would prevent accidentally breaking workflows here.

Reply:
jvmkill has been stable since its initial commit in 2015, so pointing to master seems good, since the only changes are bug fixes (like adding `_exit()` as an alternative way to die). I do like your idea of doing releases through GHA, then we could point to an exact release.